src/Commands/Component/Illuminate/Filesystem/ComponentProvider.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * This file is part of Zero Framework.
+ *
+ * (c) Nuno Maduro <enunomaduro@gmail.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace LaravelZero\Framework\Commands\Component\Illuminate\Filesystem;
+
+use LaravelZero\Framework\Commands\Component\AbstractComponentProvider;
+
+/**
+ * This is the Zero Framework illuminate/filesystem component provider class.
+ *
+ * @author Nuno Maduro <enunomaduro@gmail.com>
+ */
+class ComponentProvider extends AbstractComponentProvider
```

Comment:
There is no need to register de component on the Application.php $components attribute?

Reply:
done. Registered under the components attribute in Application class